webhook/README.md

```diff
@@ -0,0 +1,29 @@
+# Webhook configuration and scripts
+
+This folder documents the server settings needed to host JPhyloRef as a
+web server on a cluster that uses [`srun`](https://slurm.schedmd.com/srun.html) to execute programs. We use the
+[webhook](https://github.com/adnanh/webhook/) tool to set up a web server
+that accepts jobs for execution. To set this up requires the
+[`webhook` binary](https://github.com/adnanh/webhook/releases) to be
+downloaded to the same folder as these scripts.
```

Comment:
Doesn't it also require that the server on which the webhook is run be reachable from the internet, have port 80 (and/or 443?) open, _and_ be allowed to submit jobs?

In most SLURM clusters, the login node will be allowed the latter, but not the former.